pyani/scripts/subcommands/subcmd_download.py

```diff
@@ -131,7 +134,14 @@ def download_data(
                     exc_info=True,
                 )
                 skippedlist.append(
-                    Skipped(tid, uid, "", "", None, "RefSeq",)
+                    Skipped(
```

Comment:
Did `black` do this?

Reply:
Seems odd - I wouldn't have expected it. It's possible your `--line-length` setting for `black` differs from mine. (Mine is set to 88 chars).

Reply:
Is that `black`, or `flake8`? The only place I have a line like that is in `.flake8`, and it is also set to 88. I'm not sure where such things are set for `black`. The internet indicates it should be a `pyproject.toml` file, but I only have things called that in other specific projects.

Reply:
It's `black`. I have that setting in VS Code
<img width="966" alt="Screenshot 2022-03-15 at 13 21 01" src="https://user-images.githubusercontent.com/63981/158386721-8f7bf590-fade-457f-8ed8-971b19e4b8fe.png">
.